packages/salesforcedx-vscode-core/test/vscode-integration/orgBrowser/metadataCmp.test.ts

```diff
@@ -86,7 +86,7 @@ describe('get metadata components path', () => {
   function expectedPath(fileName: string) {
     return path.join(
       rootWorkspacePath,
-      '.sfdx',
```

Comment:
here instead of adding the core import I'd just stub projectPath.getMetadataDirectoryPath and use the value that stub returns here.